docs/internals/howto-release-django.txt

```diff
@@ -287,7 +352,7 @@ OK, this is the fun part, where we actually push out a release!
 
    You can check your work by running ``git tag --verify <tag>``.
 
-#. Push your work, including the tag: ``git push --tags``.
+#. Push your work (``git push``) and also the new tag (``git push --tags``).
```

Comment:
It was not clear that "push your work" and "push the tags" were two separated commands.

Reply:
```suggestion
#. Push your work and the new tag:

   .. code-block:: shell
        $ git push
        $ git push --tags
```